charts/hivemq-platform/values.schema.json

```diff
@@ -274,8 +279,15 @@
           },
           "startPriority" : {
             "type" : "integer"
+          },
+          "secretName" : {
+            "maxLength" : 253,
+            "type" : "string"
```

Comment:
Ideally, the validation rule for setting either `secretName` or `configMapName` but no both, it should be done here at the JSON schema, instead of the helper functions.
Even though, this can be done, the error message displayed to the customer is not providing any insight or useful information about what's going wrong. So I preferred to use a custom helper function instead.